hail/python/hailtop/batch/docs/docker_resources.rst

```diff
@@ -71,29 +71,26 @@ For more information about Dockerfiles and directives that can be used see the f
 Building Images
 ---------------
 
-To create a Docker image, we use a series of commands to build the image from a Dockerfile by specifying
-the context directory (in this case the current directory `.`). The `-f` option
-specifies what Dockerfile to read from. The `-t` option is the name of the image.
-More in depth information can be found `here <https://docs.docker.com/engine/reference/commandline/build/>`__.
+To create a Docker image, use `docker build <dir> -t <name> -f <dockerfile>`. <dir> specifies the context directory (typically `.`, for the current working directory).
```

Comment:
In ReStructured Text, back ticks indicate "content" which is interpreted in a context dependent way. In Python docs, "content" is rendered italicized. AFAICT, we've preferred using shell code blocks when demonstrating code examples. I think we should stick to that style.

I think something like this reads well:
> To build a Docker image, use
> ```
> docker build <dir> -t <name> -f <dockerfile>
> ```
> - ``<dir>`` is the context directory, ``.`` means the current working directory,
> - ``-t <name>`` specifies the image name, and
> - ``-f <dockerfile>`` specifies the Dockerfile file.
> 
> For example, we can build an image named ``gcr.io/my-project/my-image`` based on the Dockerfile named ``Dockerfile``, using the current working directory as the context:
> ```
> docker build . -t gcr.io/my-project/my-image -f Dockerfile
> ```
>
> In this example ...

I think it's important we stay consistent about where we place the context directory.